### DISCUSSION TITLE
DietP-Software I Linkwarden

### DISCUSSION THREAD

**Discussion by OlivierVerhoek** [MichaIng/DietPi#6903] [Software Request]
Was looking for a nicer way to manage my favorites across browsers. Instantly became my default, en enjoying a lot. Would be great if this could be added to the awesome list of software there is now.

"Linkwarden is a self-hosted, open-source collaborative bookmark manager to collect, organize and archive webpages.

The objective is to organize useful webpages and articles you find across the web in one place, and since useful webpages can go away (see the inevitability of [Link Rot](https://www.howtogeek.com/786227/what-is-link-rot-and-how-does-it-threaten-the-web/)), Linkwarden also saves a copy of each webpage as a Screenshot and PDF, ensuring accessibility even if the original content is no longer available.

Additionally, Linkwarden is designed with collaboration in mind, sharing links with the public and/or allowing multiple users to work together seamlessly." https://github.com/linkwarden/linkwarden

**Comment by kotlasu** [comment]
*gentle bump* +1 for adding Linkwarden to DietPi library.

Docker container for it exists but it’s only for arm64, so it’s useless for the older hardware (arm7/arm8).

**Comment by eclecticpassions** [comment]
I'd appreciate it dietpi-software includes linkwarden too!
